src/sqs-grep.js

```diff
@@ -252,6 +252,17 @@ class SqsGrep {
                     MessageAttributes: options.stripAttributes ? null : message.MessageAttributes,
                 }).promise();
             }
+            // Republish message to it's topic of origin
+            if (options.republish) {
+                const notification = JSON.parse(message.Body);
+                if (notification.Type === 'Notification' && notification.Message) {
```

Comment:
This line is missing a branch coverage: https://coveralls.io/builds/26481109/source?filename=src/sqs-grep.js#L258
But I don't know how that is possible to, be honest... My best guess is that it is reporting the line below:                         `MessageAttributes: options.stripAttributes ? null : message.MessageAttributes`

Perhaps we can extract this logic to a local variable and reuse it on all three locations?
```js
const attributesToUse = options.stripAttributes ? null : message.MessageAttributes;
// ...
MessageAttributes: attributesToUse
// ...
```

Reply:
I think that line is correct.

There isn't a test covering the branch were **notification.Type** is equal to 'Notification' but **notification.Message** is null.

I'll correct this.